python/google/protobuf/internal/json_format_test.py

```diff
@@ -200,6 +200,16 @@ def testExtensionToDictAndBack(self):
     json_format.ParseDict(message_dict, parsed_message)
     self.assertEqual(message, parsed_message)
 
+  def testJsonParseDictToAnyDoesNotAlterInput(self):
+    my_dict_orig = {
```

Comment:
Use orig_dict instead of my_dict_orig.
Use copied_dict instead of my_dict.

Sorry that we are a little strict on naming for better readability

Reply:
@anandolee fixed naming